lib/discovery.js

```diff
@@ -406,18 +408,12 @@ function mixinDiscovery(MySQL, mysql) {
 
   MySQL.prototype.buildPropertyType = function(columnDefinition) {
     var mysqlType = columnDefinition.dataType;
+    var columnType = columnDefinition.columnType;
     var dataLength = columnDefinition.dataLength;
 
     var type = mysqlType.toUpperCase();
     switch (type) {
       case 'CHAR':
-        if (dataLength === 1) {
-          // Treat char(1) as boolean
-          return 'Boolean';
-        } else {
-          return 'String';
-        }
-        break;
```

Comment:
I think folks use `Y`, 'N', 'T', 'F', '0', '1' for boolean too. We should keep it for backward compatibility.